include/Spectra/MatOp/SparseSymMatProd.h

```diff
@@ -66,8 +67,23 @@ class SparseSymMatProd
         MapVec y(y_out, m_mat.rows());
         y.noalias() = m_mat.template selfadjointView<Uplo>() * x;
     }
-};
 
+    ///
+    /// Perform the matrix-matrix multiplication operation \f$y=Ax\f$.
+    ///
+    SparseMatrix operator*(const SparseMatrix mat_in)
+    {
+        return m_mat * mat_in;
+    }
+
+    ///
+    /// Extract (i,j) element of the underlying matrix.
+    ///
+    Scalar operator()(Index i, Index j)
+    {
+        return m_mat(i, j);
+    }
```

Comment:
this is not correct, because we have a symmetric matrix, where parts might not be assigned. THere should be something like ``         y.noalias() = m_mat.template selfadjointView<Uplo>() * x;`` in there